<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Dialog</class>
 <widget class="QDialog" name="Dialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1178</width>
    <height>621</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Enchufe</string>
  </property>
  <widget class="QPushButton" name="back_plug_button">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>20</y>
     <width>88</width>
     <height>41</height>
    </rect>
   </property>
   <property name="text">
    <string>⤺
Atrás</string>
   </property>
  </widget>
  <widget class="QLabel" name="const_consumption_label">
   <property name="geometry">
    <rect>
     <x>500</x>
     <y>60</y>
     <width>141</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>Última lectura</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="updated_consumption_label">
   <property name="geometry">
    <rect>
     <x>530</x>
     <y>19</y>
     <width>101</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>Cargando...</string>
   </property>
   <property name="textFormat">
    <enum>Qt::AutoText</enum>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QPushButton" name="history_button">
   <property name="geometry">
    <rect>
     <x>1060</x>
     <y>570</y>
     <width>88</width>
     <height>28</height>
    </rect>
   </property>
   <property name="text">
    <string>Ver más</string>
   </property>
  </widget>
  <widget class="QPushButton" name="toggle">
   <property name="enabled">
    <bool>true</bool>
   </property>
   <property name="geometry">
    <rect>
     <x>1060</x>
     <y>20</y>
     <width>91</width>
     <height>41</height>
    </rect>
   </property>
   <property name="text">
    <string>⏻</string>
   </property>
  </widget>
  <widget class="QWidget" name="verticalLayoutWidget">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>100</y>
     <width>1121</width>
     <height>451</height>
    </rect>
   </property>
   <layout class="QVBoxLayout" name="scatter_plot_layout"/>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
